viam-cartographer/src/carto_facade/carto_facade_test.cc

```diff
@@ -444,7 +444,7 @@ BOOST_AUTO_TEST_CASE(CartoFacade_init_terminate) {
     auto path_to_internal_state = tmp_dir / fs::path("internal_state");
 
     BOOST_TEST((cf->path_to_internal_state) == path_to_internal_state.string());
-    BOOST_TEST((cf->started) == false);
+    BOOST_TEST(((cf->state) == CartoFacadeState::IO_INITIALIZED));
```

Comment:
Fun fact, you need to provide an outer paren when using boost test with enum classes: https://stackoverflow.com/questions/30860241/use-enum-classes-with-boost-test